test_util/test_installer_ccm.py

```diff
@@ -214,15 +211,10 @@ def integration_test(
         region: string indicating AWS region in which cluster is running
         agent_list: string of comma separated agent addresses
         registry_host: string for address where marathon can pull test app
-        test_minuteman: if set to True then test for minuteman service
         test_dns_search: if set to True, test for deployed mesos DNS app
         ci_flags: optional additional string to be passed to test
 
     """
-    marker_args = '-m "not minuteman"'
```

Comment:
Apparently when minuteman was made non-conditional, that was removed but setting the marker wasn't. So yea, nothing to do there, my bad